framework/rpc/grpc_testing.py

```diff
@@ -282,6 +283,41 @@ def set_not_serving(self):
         )
 
 
+class HookServiceClient(framework.rpc.grpc.GrpcClientHelper):
```

Comment:
@XuanWang-Amos FYI, now that the session drain test is in place, we should start planning on when we're going to make the hook service changes needed for the Python interop server to pass this test.

Reply:
Ack, I'll start by checking this PR.